api/v1alpha1/oidc_types.go

```diff
@@ -106,7 +106,17 @@ type OIDC struct {
 }
 
 // OIDCProvider defines the OIDC Provider configuration.
+// +kubebuilder:validation:XValidation:message="BackendRefs must be used, backendRef is not supported.",rule="!has(self.backendRef)"
```

Comment:
Even though BackendRefs is used, only one backendRef will be supported.

Reply:
Maybe add a CEL validation to make sure that `has(self.backendRefs) && size(self.backendRefs) == 1` ?

Reply:
We may support multiple services in the future. But I agree that we should make it clear that multiple providers is not supported yet.